runtime/compiler/net/ServerStream.hpp

```diff
@@ -23,19 +23,20 @@
 #ifndef SERVER_STREAM_H
 #define SERVER_STREAM_H
 
-#include <google/protobuf/io/zero_copy_stream_impl.h> 
+#include <google/protobuf/io/zero_copy_stream_impl.h>
 #include "net/ProtobufTypeConvert.hpp"
 #include "net/CommunicationStream.hpp"
 #include "env/VerboseLog.hpp"
 #include "control/Options.hpp"
+#include "control/rossa.h"
```

Comment:
Why do we need rossa.h? Ideally the communication code should not include j9 files because we may want to move this to omr in the future.

Reply:
`statusCode` is checked in `finishCompilation()` as below to decide which message type to use: `compilationCode` or `compilationFailure`. `compilationOK` is defined in `TR_CompilationErrorCode` in `control/rossa.h`. We could check it against `0` instead of `compilationOK` but it'd be a risky assumption though. 
```C++
         if (compilationOK == statusCode)
```

Reply:
I think it would be better to add a messageType to `finishCompilation()` so that `compilationOK` doesn't need to be checked inside `finishCompilation()` thus avoiding the `rossa.h` file

Reply:
Yeah, I was thinking too about the different API:
`void finishCompilation(T... args)` with no statusCode because implicitly it's going to be `compilationOK` and
`void writeError(uint32_t statusCode)` for error situations (the client has a writeError routine as well)

`finishCompilationSuccess` and `finishCompilationFailure` are good names too

Reply:
Thanks @harryyu1994, @mpirvu. Adding different APIs to handle the error path is a better choice.`finishCompilationSuccess` and `finishCompilationFailure` are good names. Adding `void writeError(uint32_t statusCode)` to `ServerStream` would make it more consistent with the `ClientStream` APIs. Will implement the latter.